### PR TITLE
[DEV-2528] Show tx fee rate [DEV-2532] Adapted to API

### PR DESCRIPTION
**Pull request by davidleomay**

### CODE REVIEW

File: src/translations/languages/de.json
Comment (Yannick1712) [inline src/translations/languages/de.json:1]: I think we have to rename the Description field + value to message or?

File: src/translations/languages/fr.json
Comment (Yannick1712) [inline src/translations/languages/fr.json:1]: Same here

File: src/translations/languages/it.json
Comment (Yannick1712) [inline src/translations/languages/it.json:1]: Same here

File: src/translations/languages/de.json
Comment (davidleomay) [reply src/translations/languages/de.json:1]: The translated string is still `Description`:
image
Do you think we should rename it to `Message`?

Comment (Yannick1712) [reply src/translations/languages/de.json:1]: Hm yeah we could leave it like that. Don't know first thought I think I would prefer message but we can also take the description 👍

**Pull request merged**